pandas/core/array_algos/take.py

```diff
@@ -353,31 +354,6 @@ def wrapper(
     return wrapper
 
 
-_take_1d_dict = {
-    ("int8", "int8"): libalgos.take_1d_int8_int8,
-    ("int8", "int32"): libalgos.take_1d_int8_int32,
-    ("int8", "int64"): libalgos.take_1d_int8_int64,
-    ("int8", "float64"): libalgos.take_1d_int8_float64,
-    ("int16", "int16"): libalgos.take_1d_int16_int16,
-    ("int16", "int32"): libalgos.take_1d_int16_int32,
-    ("int16", "int64"): libalgos.take_1d_int16_int64,
-    ("int16", "float64"): libalgos.take_1d_int16_float64,
-    ("int32", "int32"): libalgos.take_1d_int32_int32,
-    ("int32", "int64"): libalgos.take_1d_int32_int64,
-    ("int32", "float64"): libalgos.take_1d_int32_float64,
-    ("int64", "int64"): libalgos.take_1d_int64_int64,
-    ("int64", "float64"): libalgos.take_1d_int64_float64,
-    ("float32", "float32"): libalgos.take_1d_float32_float32,
-    ("float32", "float64"): libalgos.take_1d_float32_float64,
-    ("float64", "float64"): libalgos.take_1d_float64_float64,
-    ("object", "object"): libalgos.take_1d_object_object,
-    ("bool", "bool"): _view_wrapper(libalgos.take_1d_bool_bool, np.uint8, np.uint8),
-    ("bool", "object"): _view_wrapper(libalgos.take_1d_bool_object, np.uint8, None),
-    ("datetime64[ns]", "datetime64[ns]"): _view_wrapper(
-        libalgos.take_1d_int64_int64, np.int64, np.int64, np.int64
-    ),
-}
-
```

Comment:
have a commit ready to push that reverts this change to the main code base.. the jitted python code is so messy, especially trying to add specializations so that could match the cython signatures. The upside is that take_1d is ideal parallel candidate. Long term, if paralellization is performed in a higher level loop, i.e. columns in a DataFrame, then take would be serialized, but for now with the scope limited to a libs drop-in, then will parallelize at the libs level to keep the main codebase unchanged.